package/MDAnalysis/topology/TXYZParser.py

```diff
@@ -69,8 +72,11 @@ class TXYZParser(TopologyReaderBase):
 
     - Atomnames
     - Atomtypes
+    - Elements
```

Comment:
```suggestion
    - Elements (if all atom names are element symbols)
```

Otherwise users will be surprised when they do not have an `elements` attribute.